diff --git a/profiler/src/main.cpp b/profiler/src/main.cpp
index 679e0ec0..defa6113 100644
--- a/profiler/src/main.cpp
+++ b/profiler/src/main.cpp
@@ -281,7 +281,7 @@ int main( int argc, char** argv )
 
     std::unique_ptr<tracy::FileRead> initFileOpen;
 #ifdef __EMSCRIPTEN__
-    initFileOpen = std::unique_ptr<tracy::FileRead>( tracy::FileRead::Open( "embed.tracy" ) );
+    // initFileOpen = std::unique_ptr<tracy::FileRead>( tracy::FileRead::Open( "embed.tracy" ) );
 #endif
     if( argc == 2 )
     {
@@ -903,6 +903,7 @@ static void DrawContents()
         {
             tracy::OpenWebpage( "https://github.com/sponsors/wolfpld/" );
         }
+#ifndef __EMSCRIPTEN__
         if( updateVersion > tracy::FileVersion( tracy::Version::Major, tracy::Version::Minor, tracy::Version::Patch ) )
         {
             ImGui::Separator();
@@ -923,6 +924,7 @@ static void DrawContents()
                 }
             }
         }
+#endif
         if( s_isElevated )
         {
             ImGui::Separator();
diff --git a/profiler/wasm/index.html b/profiler/wasm/index.html
index 88a0712d..8a08ae4d 100644
--- a/profiler/wasm/index.html
+++ b/profiler/wasm/index.html
@@ -138,7 +138,7 @@
           Module.setStatus(left ? 'Preparing... (' + (this.totalDependencies-left) + '/' + this.totalDependencies + ')' : 'All downloads complete.');
         }
       };
-      Module.setStatus('Downloading...');
+      // Module.setStatus('Downloading...');
       window.onerror = function(event) {
         // TODO: do not warn on ok events like simulating an infinite loop or exitStatus
         Module.setStatus('Exception thrown, see JavaScript console');
